_includes/download/board.html

```diff
@@ -160,7 +160,61 @@ <h3>Past Releases</h3>
 {% assign bootloader_id = bootloader_board.bootloader_id %}
 {% if bootloader_version and bootloader_id and bootloader_board %}
 <div class="section unrecommended">
-  <h3>UF2 Bootloader</h3>
+  <h3>Install/Repair UF2 Bootloader</h3>
+  <p>
+    Latest version: {{ bootloader_version }}
+  </p>
+  <p>
+    <strong>The bootloader allows you to load CircuitPython, Makecode, and Arduino programs.
+      The bootloader is not CircuitPython.</strong>
+  </p>
+  <p>
+    It is not necessary to reinstall the bootloader unless a BOOT drive is not visible when in UF2 bootloader mode (FTHRS2BOOT, MAGTAGBOOT, HOUSEBOOT, etc.), in which case the UF2 bootloader may need to be repaired.
+  </p>
+  <p>
+    If a UF2 bootloader has never been installed on the board, or the UF2 bootloader was overwritten by erasing or overwriting the flash, the UF2 bootloader should be installed in order to flash .uf2 files onto the board.
+  </p>
```

Comment:
Could you move this inside the `esp32s2` section? On non-ESP32S2 boards, there is always a UF2 bootloader (except in very special circumstances), so I think this is a little confusing for other board families

Reply:
Fixed in e4474dfa5a56a46ffff976f758d955d20a786572